README.md

```diff
@@ -384,7 +384,7 @@ Seonmyeong Bak, Karthik Senthil, Juan Galvez, Michael Robson,
 Raghavendra Kanakagiri, Venkatasubrahmanian Narayanan. Nitin Bhat, and
 Justin Szaday. Current developers include: Eric Bohm, Ronak Buch, Eric
 Mikida, Sam White, Kavitha Chandrasekar, Jaemin Choi, Matthias Diener,
-Evan Ramos, Zane Fink, Pathikrit Ghosh, Maya Taylor, Aditya Bhosale,
```

Comment:
Typo introduced: "Bhosa" should be "Bhosale."

Reply:
Good spot.  Fixed.